finagle-http/src/test/scala/com/twitter/finagle/http/HttpStreamingEndToEndTest.scala

```diff
@@ -0,0 +1,35 @@
+package com.twitter.finagle.http
+
+import com.twitter.conversions.time._
+import com.twitter.finagle.{Http => _Http, Service}
+import com.twitter.io.{Buf, Reader}
+import com.twitter.util.{Await, Future, JavaTimer}
+import org.scalatest.FunSuite
+import java.net.InetSocketAddress
+
+class HttpStreamingEndToEndTest extends FunSuite {
```

Comment:
What do you think of adding this to [StreamingTest.scala](https://github.com/twitter/finagle/blob/develop/finagle-http/src/test/scala/com/twitter/finagle/http/StreamingTest.scala)?